public/index.html

```diff
@@ -2,8 +2,15 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Webpack App</title>
+    <title>Headlines News App</title>
+    <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,700" rel="stylesheet"> 
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">    
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link href="/css/main.css" rel="stylesheet">
   </head>
   <body>
-  <script type="text/javascript" src="bundle.min.js"></script></body>
+    <div id="root"></div>
+  <script type="text/javascript" src="/js/bundle.min.js"></script>
```

Comment:
I won't be pushing the bundle.min.js file. Heroku bundles it using webpack when the app is deployed